build.sbt

```diff
@@ -22,16 +22,10 @@ testFrameworks += new TestFramework("zio.test.sbt.ZTestFramework")
 
 Global / cancelable := false
 
-dockerBaseImage := "gcr.io/distroless/java:8"
-
 dockerUpdateLatest := true
-
-dockerAutoremoveMultiStageIntermediateImages := false
-
-dockerCommands := Seq(
-  Cmd("FROM", "gcr.io/distroless/java:8"),
-  Cmd("WORKDIR", "/opt/docker"),
-  Cmd("COPY", "1/opt", "/opt"),
-  Cmd("COPY", "2/opt", "/opt"),
-  ExecCmd("CMD", s"/opt/docker/lib/${(artifactPath in packageJavaLauncherJar).value.getName}")
-)
+dockerBaseImage := "gcr.io/distroless/java:11"
+daemonUserUid in Docker := None
+daemonUser in Docker := "root"
+dockerPermissionStrategy := DockerPermissionStrategy.None
+dockerEntrypoint := Seq("java", "-jar")
+dockerCmd :=  Seq(s"/opt/docker/lib/${(artifactPath in packageJavaLauncherJar).value.getName}")
```

Comment:
alternative
```
dockerEntrypoint := Seq("java", "-jar",s"/opt/docker/lib/${(artifactPath in packageJavaLauncherJar).value.getName}")
dockerCmd :=  Seq("")
```
to be able to use args
`docker run -p 8080:8080 hello-uzhttp:latest arg1 arg2 arg3`